musictheorpy/notes.py

```diff
@@ -63,6 +63,21 @@ def descend_interval(self, qualified_interval_name):
             raise NoteNameError("Descending a %s from %s results in an invalid note." % (qualified_interval_name,
                                                                                          self._qualified_name)) from None
 
+    def find_interval_from_root(self, top_note):
+        """
+        :param top_note: a string that is verified as a valid Note and passed to the
+        interval_builder.find_interval_from_root method to find its relation to root note
+        :return <str> interval: the interval at which the top note is in relation from the root
+        """
+        root_note = self.qualified_name
+        try:
+            note_name = Note(top_note).qualified_name
```

Comment:
I don't believe this line is necessary. It looks like you are making a Note object just to get the qualified name. This is equivalent to just using ``top_note`` directly.

Reply:
Since the user isnt required to send in a valid note, I thought that by attempting to parse it as a note, it would act as validation. If you still feel that that is redundant ill change it, but that was the mindset.

Reply:
Disregard. I see the light and have removed it.